mava/systems/tf/maddpg/system.py

```diff
@@ -270,6 +282,33 @@ def replay(self) -> Any:
 
         return self._builder.make_replay_tables(self._environment_spec)
 
+    def create_system(
```

Comment:
This is great 😄  We should also do this for the other systems.

Reply:
Agreed. It looks nice. I added it to #246 :+1: